### ISSUE TITLE
[a11y]: FileUploader [default variant] - Doesn't annouce uploading status

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon#14454]
### Package

@carbon/react

### Package version

@carbon/react@1.35.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

No indication for screen reader when file is uploaded
Screen reader needs to announce that file is being uploaded and then uploading is complete 
No indication for delete button

*Note: drag and drop fileuploader DOES announce correctly

### Reproduction/example

https://react.carbondesignsystem.com/iframe.html?args=&id=components-fileuploader--default&viewMode=story

### Steps to reproduce

https://react.carbondesignsystem.com/iframe.html?args=&id=components-fileuploader--default&viewMode=story

Upload a file,notice that it doesn't announce that a file is being uploaded or uploading complete. 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by sstrubberg** [comment]
This is a storybook problem, not a component problem.
